corehq/apps/callcenter/utils.py

```diff
@@ -52,68 +106,64 @@ def sync_user_case(commcare_user, case_type, owner_id, case=None):
     """
     with CriticalSection(['user_case_%s_for_%s' % (case_type, commcare_user._id)]):
         domain = commcare_user.project
+        fields = _get_user_case_fields(commcare_user)
+        case = case or get_case_by_domain_hq_user_id(domain.name, commcare_user._id, case_type)
+        close = commcare_user.to_be_deleted() or not commcare_user.is_active
```

Comment:
Does this mean that when a user is deleted, their user case is closed, not marked as deleted? Is this correct? (It is different from the behaviour of normal cases.)